code/lib/cli/src/versions.ts

```diff
@@ -68,6 +68,7 @@ export default {
   '@storybook/source-loader': '7.0.0-alpha.54',
   '@storybook/store': '7.0.0-alpha.54',
   '@storybook/svelte': '7.0.0-alpha.54',
+  '@storybook/svelte-kit': '7.0.0-alpha.54',
```

Comment:
@shilman I don't know if this is legal to do. Or even if there's something other we need to do because this package is yet to be released.